src/test/include.am

```diff
@@ -79,7 +79,14 @@ src_test_AM_CPPFLAGS = -DSHARE_DATADIR="\"$(datadir)\"" \
 # This seems to matter nowhere but on Windows, but I assure you that it
 # matters a lot there, and is quite hard to debug if you forget to do it.
 
+DIRAUTH_MOD = \
+  src/or/dirauth/dircollate.c				\
```

Comment:
Indentation seems off here? Should be \t.

Reply:
Yeah that commit doesn't work and will disappear. Sorry about this, it got forgotten :(